tests/implementor/test_impl_run.py

```diff
@@ -148,7 +148,13 @@ def test_get_run(self, token, empty_simulation_configuration, monkeypatch):
         fetched_run, status = impl.run.get_run({"identifier": run.id}, token)
         assert status == 200
         assert fetched_run["state"] == "PROGRESS" or fetched_run["state"] == "SUCCESS"
-        assert fetched_run["progress"] > 0
+        assert isinstance(fetched_run["progress"], int)
+        assert isinstance(fetched_run["id"], str)
+        assert isinstance(fetched_run["readable_id"], str)
+        assert isinstance(fetched_run["updated_at"], str)
+        assert isinstance(fetched_run["created_at"], str)
+        assert isinstance(fetched_run["process_id"], str)
+        assert isinstance(fetched_run["simulation"], str)
```

Comment:
This is supposed to be `simulation` and not `simulation_id`?

Reply:
`simulation` is correct. It's the return value for the user 😇